vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -88,6 +94,25 @@ public WebElement clearFieldLocatedByUsingKeyboard(SearchAttributes locator)
         return element;
     }
 
+    /**
+     * Checks that all <b>elements</b> sorted by text with using <b>ruleForSorting</b>
+     * @param ruleForSorting The set of sort's rule<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ALPHABETICALLY</b> - will check that list sorting alphabetically,
+     * <li><b>REVERSE_ALPHABETICALLY</b> - will check that list sorting reverse alphabetically
+     * </ul>
+     * @param elements list of elements
+     */
+    @Then("all '$elements' sorted $ruleForSorting by text")
```

Comment:
```
Then elements located `$locator` are sorted by text in $sortingOrder order")
```
where `$sortingOrder ` is `ascending` or `descending`

Reply:
@valfirst why not save a list of the text values and compare it with a separate step